.github/workflows/generate.yml

```diff
@@ -0,0 +1,49 @@
+name: Code Generation
+
+on:
+  push:
+    branches: ["main"]
+
+  workflow_dispatch:
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: frenck/action-setup-yq@v1
+      - name: Downgrade OAS
+        run: bash scripts/downgrade-oas-version.sh
+  build:
+    needs: [ setup ]
+    runs-on: ubuntu-latest
+    strategy:
+            matrix:
+                target:
+                    - go-gin-server
+                    - python-flask 
+                cx-version:
+                    # downgradePath represents the spec file path as created as 
+                    # part of scripts/downgrade-oas-version.sh in a previous job 
+                    - {semantic: "5.0", downgradePath: "5dot0"}
+                    - {semantic: "4.6", downgradePath: "4dot6"}
```

Comment:
Does this mean we need to update it with every new version? We merged 5.1 yesterday.

Reply:
I dont see that version here, where was it merged? https://github.com/plaid/core-exchange/tree/main/versions